### DISCUSSION TITLE
Kaynak dosyası "stdio.h" açılamıyor ; Path Yolu bulunamadı. Hatası

### DISCUSSION THREAD

**Discussion by ibrahimeth** [19/classroom#26] [C] [answered]
Hayırlı çalışmalar hocam ,

Vs code üzerinden

`gcc  filename.c -o filenam `

komutunu kullandığım da 

```
gcc : The term 'gcc' is not recognized as the name of a cmdlet, function, script file, or operable program. Check the spelling of the name, or if a path was included, verify that the path is correct and tr 
y again. 
```
hatası alıyorum aynı zamanda 
c_cpp_properties.json dosyasında  sarı vurgulu kısımda "\\?\C:\Classroom\rootfs\usr/bin/gcc" bulunamıyor. Hatası almaktayım.

![1](https://user-images.githubusercontent.com/92324388/158159813-03d75b12-bfaa-4346-a776-55a4370ca2ae.png)

ayrıca eklemek istediğim Microsoft C/C++ Uzantısı sayesinde Derleyici yolunda sıkıntı yaşadığımı öğrendim 
form ve Alaturka Classroom Terminal SSS kısmını da gözden geçirdim
bu konuda araştırmalarım sonucunda ulaştığım MinGW programını mı indirmeliyim size danışmak istedim .

SAYGILARIMLA.

![2](https://user-images.githubusercontent.com/92324388/158159818-71148dd1-b8dc-4334-98a0-bb705eba34f0.png)

**Comment by ibrahimeth** [comment, marked as answer]
Sorunun daha detaylı araştırmalarım sonucu Çözümünün Ekteki şeklinde yapılabildiğini öğrendim . Diğer arkadaşlara umarım yol gösterir.

![1ii](https://user-images.githubusercontent.com/92324388/158164288-d06eeb2d-db62-4cc9-a2d6-fd47d84e5253.png)
![2ii](https://user-images.githubusercontent.com/92324388/158164293-b34e8505-f54f-4227-9dbd-d5c123bcdc9a.png)
![4ii](https://user-images.githubusercontent.com/92324388/158164300-7d0e1b1c-a25a-4e76-b66a-aa129576e0da.png)

**Comment by roktas** [comment]
@ibrahimeth https://www.youtube.com/embed/e17yzH8eDqE

https://classroom.alaturka.dev/guide/#vs-code-icinden-wsl-e-erisim

**Reply by ibrahimeth** [reply]
Teşekkür ederim hocam
